plugins/woocommerce/src/Packages.php

```diff
@@ -74,6 +94,61 @@ public static function package_exists( $package ) {
 		return file_exists( dirname( __DIR__ ) . '/packages/' . $package );
 	}
 
+	/**
+	 * Checks a package exists by looking for it's directory.
+	 *
+	 * @param string $string Class name.
+	 * @return boolean
+	 */
+	public static function should_load_class( $class ) {
+
+		foreach( self::$merged_packages as $merged_package_name => $merged_package_class ) {
+			if ( $class === str_replace( 'woocommerce-', 'wc_', $merged_package_name ) ) {
+				return true;
+			}
+		}
+
+		return false;
+	}
+
+	/**
+	 * Gets all merged, enabled packages.
+	 *
+	 * @return array
+	 */
+	protected static function get_enabled_packages() {
+		$enabled_packages = array();
+
+		foreach( self::$merged_packages as $merged_package_name => $package_class ) {
+
+			// For gradual rollouts, ensure that a package is enabled for user's remote variant number.
+			$experimental_package_enabled = method_exists( $package_class, 'is_enabled' ) ?
```

Comment:
optional nitpick, but maybe it makes more sense to have the default as `false`. we do want packages to have `is_enabled` method.

Reply:
I was thinking that perhaps for feature packages we won't want to do a gradual rollout, but let's switch to a `false` default value for now and re-visit if this becomes an issue in the future.